cli/crates/production-server/src/config.rs

```diff
@@ -107,7 +107,7 @@ pub struct TlsConfig {
     pub key: PathBuf,
 }
 
-#[derive(Debug, PartialEq, serde::Deserialize, Default)]
+#[derive(Debug, PartialEq, serde::Deserialize, Default, Clone)]
 #[serde(deny_unknown_fields)]
 pub struct TrustedDocumentsConfig {
```

Comment:
bypass_header_name & value should be merged into a tuple imho it they only make sense together

Reply:
ah I guess that's the toml config. Should be a struct with `#[serde(flatten)]` then

Reply:
done